tools/tf2openapi/generator/testdata/TestColFmtMultipleTensors.golden

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
.golden.json will help editors highlight this correctly.

Reply:
done